### PR TITLE
Added the fake data to generate car brands names

### PR DESCRIPTION
**Pull request by felipeflorencio**
Hello I would like to contribute adding the generated data for Car Brands it's update as far I saw, they have tesla so looks really update ;) 

Any change please let me know 👍

### CODE REVIEW

File: Tests/Fakery/Generators/CarSpec.swift [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        cars = Car(parser: parser)
+      }
+      
+      describe("#brand") {
+        it("returns the correct brand name text") {
+          let name = cars.brand()
+          expect(name).to(equal("BMW"))
+        }
+      }
+
```
Comment (dunyakirkali) [inline Tests/Fakery/Generators/CarSpec.swift]: @felipeflorencio maybe remove this line?

**Comment by dunyakirkali** [comment]
@jeroendevrind Could you also update the `README.md` with some usage instructions?

File: Tests/Fakery/Generators/CarSpec.swift (hunk 1 above)
Comment (felipeflorencio) [reply Tests/Fakery/Generators/CarSpec.swift]: Adjusted :) 👍

**Comment by felipeflorencio** [comment]
> @felipeflorencio Could you also update the `README.md` with some usage instructions?

Added 👍

**Comment by vadymmarkov** [comment]
Great @felipeflorencio. Could you also add this generator to https://github.com/vadymmarkov/Fakery/blob/master/Sources/Fakery/Faker.swift to make it accessible.

**Comment by felipeflorencio** [comment]
> Great @felipeflorencio. Could you also add this generator to https://github.com/vadymmarkov/Fakery/blob/master/Sources/Fakery/Faker.swift to make it accessible.

Done 👍

**Comment by felipeflorencio** [comment]
All conflicts are fixed now, a lot of pull request's 💯

**Comment by dunyakirkali** [comment]
@vadymmarkov ☝️

**Pull request merged**